cpp/velox/compute/VeloxBackend.h

```diff
@@ -69,6 +69,10 @@ class VeloxBackend final : public Backend {
     return std::make_shared<VeloxParquetDatasource>(filePath, fileName, schema);
   }
 
+  int64_t spillFixedSize(const std::string& taskTag, int64_t size) override;
```

Comment:
Will it be possible to add the API to result iterator (which is bound to a Velox task) rather than the backend instance? If that doesn't require the arch code to be changed (not sure about it)?